src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2659,7 +2660,7 @@ namespace Akka.Event
         public Debug(string logSource, System.Type logClass, object message) { }
         public override Akka.Event.LogLevel LogLevel() { }
     }
-    public class DefaultLogger : Akka.Actor.ActorBase
+    public class DefaultLogger : Akka.Actor.ActorBase, Akka.Dispatch.IRequiresMessageQueue<Akka.Event.ILoggerMessageQueueSemantics>
```

Comment:
cc @Danthar @rogeralsing hmmm... think we'll need to update the logging plugins if we add this? Or should this just work automatically since it doesn't really affect the public API?